designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/FEStatisticsRepository.scala

```diff
@@ -0,0 +1,9 @@
+package pl.touk.nussknacker.ui.db.timeseries
+
+import scala.concurrent.ExecutionContext
+import scala.language.higherKinds
+
+trait FEStatisticsRepository[F[_]] {
+  def write(statistics: Map[String, Long])(implicit ec: ExecutionContext): F[Unit]
```

Comment:
the `ExecutionContext` is strictly related to `Future`. We declare a generic F as an effect, so let's move the execution context to the repository implementation (at the constructor level)